terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/kubeconfig

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```

Comment:
Can we get rid of this file?

It's a pain to keep it up to date and should always be available through `terraform output`.

Reply:
sure, I did wonder about this, will delete

Reply:
done